### ISSUE TITLE
[Crafting] Some crafting components are missing info like price or weight

### ISSUE THREAD

**Issue by InnilToo** [Mythacri/Mythacri-Foundry-Scripts#197]
I could not find the list to provide exact items.

**Comment by InnilToo** [comment]
https://github.com/Mythacri/Mythacri-Foundry-Scripts/issues/210
https://github.com/Mythacri/Mythacri-Foundry-Scripts/issues/209

**Comment by krbz999** [comment]
Are they actually meant to have price and/or weight?

**Comment by Mythacri** [comment]
Yes. I need to account for what it should logically be in marbles

**Comment by InnilToo** [comment]
> Are they actually meant to have price and/or weight?

Others do, so it's an inconsistency. Either way is good with me as long as it's same everywhere.
